core/templates/domain/state_card/state-card.model.ts

```diff
@@ -161,7 +174,7 @@ export class StateCard {
     let interactionId = this.getInteractionId();
     return (
       !interactionId ||
-        INTERACTION_SPECS[interactionId].display_mode ===
+        INTERACTION_SPECS[<InteractionSpecsKey>interactionId].display_mode ===
```

Comment:
Move this (`INTERACTION_SPECS[<InteractionSpecsKey>interactionId].display_mode`) to a variable so that the code is more readable.